scripts/linux/build-appimage.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+set -e
+set -x
+
+# use RAM disk if possible
+if [ "$CI" == "" ] && [ -d /dev/shm ]; then
+    TEMP_BASE=/dev/shm
+else
+    TEMP_BASE=/tmp
+fi
+
+BUILD_DIR=$(mktemp -d -p "$TEMP_BASE" OpenRCT2-appimage-build-XXXXXX)
+
+cleanup () {
+    if [ -d "$BUILD_DIR" ]; then
+        rm -rf "$BUILD_DIR"
+    fi
+}
+
+[ "$NO_CLEANUP" == "" ] && trap cleanup EXIT
+
+# store repo root as variable
+REPO_ROOT=$(readlink -f $(dirname "$0")/../..)
+OLD_CWD=$(readlink -f .)
+
+pushd "$BUILD_DIR"
+
+# standard linuxdeploy pattern
+#see https://docs.appimage.org/packaging-guide/from-source/index.html for more information
+cmake "$REPO_ROOT" -DCMAKE_INSTALL_PREFIX=/usr
+
+make -j$(nproc) VERBOSE=1
+make install DESTDIR=AppDir
```

Comment:
Please use `ninja` instead

Reply:
As long as `ninja` supports the `DESTDIR` part, that should not be possible. I'm quite sure it doesn't, though.

Reply:
There's no problem using `DESTDIR` with `ninja`.

Reply:
```
+ ninja -j8
[409/409] Linking CXX executable openrct2
+ ninja install DESTDIR=AppDir
ninja: error: unknown target 'DESTDIR=AppDir'
```

Please provide an example how it should be used, or edit the file directly.

Reply:
You're using it wrong. It's an environment variable, so:
`$ DESTDIR=AppDir ninja install`